packages/opine/opine.0.1.3/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Python AST unparse implementation in OCaml"
+description:
+  "Opine is an OCaml library that unparses the python AST back to python code."
+maintainer: ["arulselvan1234@gmail.com"]
+authors: ["Arulselvan Madhavan"]
+license: "MIT"
+tags: ["topics" "ocaml" "python" "ast"]
+homepage: "https://github.com/ArulselvanMadhavan/opine"
+bug-reports: "https://github.com/ArulselvanMadhavan/opine/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.6"}
+  "ppx_hash"
+  "ppx_compare"
+  "ppx_sexp_conv"
+  "ppx_make"
+  "ppx_deriving"
+  "pyre-ast" {>= "0.6.0"}
```

Comment:
No version 0.6.0 of pyre-ast exist. Did you mean 0.1.6 or something?

Reply:
I'm sorry about that. Yes, I did intend to write 0.1.6. Fixed it!